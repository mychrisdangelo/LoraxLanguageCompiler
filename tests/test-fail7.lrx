/* Tests type mismatch of actual parameters */
int function(int a, int b)
{
	print(a + b);
}
int main () {
	
	function(9, 7.0);
	
}