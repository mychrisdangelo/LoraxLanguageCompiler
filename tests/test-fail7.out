Fatal error: exception Failure("function function's argument types don't match its formals")
